### DISCUSSION TITLE
Idea: Make `submodules: true` the default for Bundler 3

### DISCUSSION THREAD

**Discussion by ankane** [ruby/rubygems#4902] [Ideas]
Hi, I wanted to get thoughts on making `submodules: true` the default for Bundler 3 for `git` (and related) sources.

Pro: Repos with submodules should just work.
Con: May unnecessarily download unneeded submodules.

**Comment by deivid-rodriguez** [comment]
In my opinion, this makes sense as a better default for Bundler 3 👍.
